adding full table

awanafiaz opened the pull request:
change all 6 rmds

awanafiaz committed:
remove table warnings

change all 6 rmds

awanafiaz committed:
added All Tables

Added all tables, fixed column specifications, removed table print warnings.

awanafiaz committed:
add modified data files

4 csv's for general, omics, imaging, and clinical

carriewright11 commented on zzz_allTables.Rmd:
```diff
@@ -0,0 +1,79 @@
+
+### **Overview**
+
+The following table contains a list of all ITCR-funded as well as some other useful bioinformatics platforms, including general informatics and biomedical, imaging, omics, and clinical platforms that can help researchers work with publicly available data or their own data and provide computing resource support across different areas of bioinformatics research.
```
```suggestion 
The following table contains a list of ITCR-funded bioinformatics platforms, as well as some other useful platforms, including those that allow users to work with a variety of types of data including imaging, omics, and clinical data. These resources can help researchers work with publicly available data or their own data and provide computing support across different areas of bioinformatics research.
```

carriewright11 replied:
This technically doesn't have **all** itcr -funded tools  (some are too new or out-of-date)

awanafiaz replied:
Gotcha. I had to manually include these changes to the new files (renamed) and committed those to this PR (as these are outdated files).

carriewright11 commented:
Let's discuss this one at the meeting - good job getting rid of the warnings

awanafiaz committed:
Corrected All tables, fixed data file update issue

No need to use arbitrary names for Rmd knitting hierarchy.

awanafiaz committed:
adding feedback and overview pages

- rename all tables to all platforms
- place all tables at last
- added overview page will all definitions
-  added feedback page with relevant questions
- included the pricing information on TOPAS with link
- Revamped the contact info page
- fixed other minor issues

awanafiaz committed:
added new word exceptions

carriewright11 commented on generalTable.Rmd:
```diff
@@ -14,7 +14,7 @@ output: html_document
 
 The following table contains a list of general informatics and biomedical platforms that can help researchers work with publicly available data and or their own data. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data, and provide computing resource support across different areas of bioinformatics research. 
```
```suggestion
The following table contains a list of informatics and biomedical platforms that can help researchers work with **multiple types** of publicly available data and or their own data. These platforms provide computing resource support across different areas of bioinformatics research. 
```

awanafiaz replied:
Same here as well. This is now called multiTable.Rmd

carriewright11 commented on zzz_allTables.Rmd:
```diff
@@ -0,0 +1,79 @@
+---
+title: "**All Tables**"
```
Does this file still need to be called `zzz_`?

awanafiaz replied:
No. I changed its name to allTables.Rmd. I think it is now showing outdated on github.

carriewright11 reviewed (commented):
Looks like a great start! I have some suggested changes.

carriewright11 commented on platformsOverview.Rmd:
```diff
@@ -0,0 +1,38 @@
+
+### **Overview**
+
+This resources on this website aims to aid cancer researchers find bioinformatics computing platforms and data resources, many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Here is the glossary for relevant terms used in the website:
```
```suggestion
This resources aims to improve access to bioinformatics computing platforms many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Here is the glossary for relevant terms used in these tables:
```

carriewright11 commented on platformsOverview.Rmd:
```diff
@@ -0,0 +1,38 @@
+  
+  - **Clinical** platforms work with several types of clinical data, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms.
+  
+  - **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data containing genomics, proteomics, imaging (histology and radiology), as well as other various types of data.
```
This reminds me that we might want to change the names for the RMD files so that the links are more informative. I can do that as it is a bit tricky, but we will want to link to that afterward.

awanafiaz committed:
Update platformsOverview.Rmd

Co-authored-by: Carrie Wright <23014755+carriewright11@users.noreply.github.com>

awanafiaz committed:
updated text description

I had to manually include these changes to the new files (renamed) and committed those to this PR (as these are outdated files).

awanafiaz commented:
Everything should be all OK now and ready for merging.

carriewright11 commented on platformsOverview.Rmd:
```diff
@@ -0,0 +1,38 @@
+
+This resources aims to improve access to bioinformatics computing platforms many of which are funded by the ITCR (Informatics Technology for Cancer Research) program of the NCI (National Cancer Institute). Here is the glossary for relevant terms used in these tables:
+
+  - **Platforms** are web or cloud-based websites/webpages that host computational tools or data resources.  
```
```suggestion
  - **Platforms** are web or cloud-based websites/webpages that allow users to analyze their own data and/or publicly available data.
```

carriewright11 commented on platformsOverview.Rmd:
```diff
@@ -0,0 +1,38 @@
+
+  - **Platforms** are web or cloud-based websites/webpages that host computational tools or data resources.  
+  
+  - **Multi-data type platforms** work with a wide variety of data types, from imaging, histological, omics, and clinical data. 
```
```suggestion
  - **Multi-data type platforms**allow users to work with more than one type of data, including a wide variety of data types, from imaging, histological, omics, and clinical data. 
  ```

carriewright11 commented on platformsOverview.Rmd:
```diff
@@ -0,0 +1,38 @@
+  
+  - **Clinical** platforms work with several types of clinical data, such as EMR Text and Clinical Notes, utilizing Natural Language Processing techniques, AI and machine learning algorithms.
+  
+  - **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data containing genomics, proteomics, imaging (histology and radiology), as well as other various types of data.
```
```suggestion
  - **Data resources** are remotely hosted sources of curated or collaborative bioinformatics data containing genomics, proteomics, imaging (histology and radiology), as well as other various types of data. See the Data Resource page for more information.
  ```

carriewright11 reviewed (commented):
Looks good once we incorporate some text changes that I suggested

carriewright11 committed:
Update platformsOverview.Rmd

carriewright11 committed:
Update platformsOverview.Rmd

carriewright11 committed:
Update platformsOverview.Rmd
